app/src/main/java/com/example/waterme/ui/WaterMeApp.kt

```diff
@@ -75,31 +79,45 @@ fun PlantListContent(
     onScheduleReminder: (Reminder) -> Unit,
     modifier: Modifier = Modifier
 ) {
-    var selectedPlant by rememberSaveable { mutableStateOf(plants[0]) }
-    var showReminderDialog by rememberSaveable { mutableStateOf(false) }
-    LazyColumn(
-        contentPadding = PaddingValues(dimensionResource(id = R.dimen.padding_medium)),
-        verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.padding_medium)),
-        modifier = modifier
-    ) {
-        items(items = plants) {
-            PlantListItem(
-                plant = it,
-                onItemSelect = { plant ->
-                    selectedPlant = plant
-                    showReminderDialog = true
-                },
-                modifier = Modifier
-                    .fillMaxWidth()
+    Scaffold {
+        var selectedPlant by rememberSaveable { mutableStateOf(plants[0]) }
+        var showReminderDialog by rememberSaveable { mutableStateOf(false) }
+        val layoutDirection = LocalLayoutDirection.current
+        LazyColumn(
+            verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.padding_medium)),
+            modifier = modifier
+                .statusBarsPadding()
+                .padding(
+                    start = dimensionResource(id = R.dimen.padding_medium),
+                    end = it.calculateEndPadding(layoutDirection)
+                            + dimensionResource(id = R.dimen.padding_medium)
+                ),
+            contentPadding = PaddingValues(
+                start = 0.dp,
+                top = dimensionResource(id = R.dimen.padding_medium),
+                end = 0.dp,
+                bottom = 0.dp
+            )
+        ) {
```

Comment:
I am extremely sorry to miss this in the first review, 
Why are we not setting `innerpadding` for `contentPadding` of `LazyColumn`? 
The way we did in other apps, like **Woof**. 

```
Scaffold(
        topBar = {
            WoofTopAppBar()
        }
    ) { it ->
        LazyColumn(contentPadding = it) {
            items(dogs) {
                DogItem(
                    dog = it,
                    modifier = Modifier.padding(dimensionResource(R.dimen.padding_small))
                )
            }
        }
    }
```